pandas/tests/copy_view/test_interp_fillna.py

```diff
@@ -146,7 +146,7 @@ def test_interpolate_downcast_reference_triggers_copy():
 
     msg = "Can not interpolate with method=pad"
     with pytest.raises(ValueError, match=msg):
-        df.interpolate(method="pad", inplace=True, downcast="infer")
```

Comment:
It appears that a lot of interpolate usages are still modified in the PR. _Only_ tests that used to call`.resample(...).interpolate(interpolate=)` should have been changed